.github/CHANGELOG.md

```diff
@@ -1054,3 +1054,4 @@
 1. [CDU] Full +/- button functionality - @lhoenig (Lukas Hoenig)
 1. [DCDU] Fixed MSG- and MSG+ button labels - @tyler58546 (tyler58546)
 1. [OVHD] Fixed fire push button not being able to be stowed - Julian Sebline (Julian Sebline#8476 on Discord)
+1. [RMP] Fixed the behavior of the RMPs and ACPs (VHFs only) - Julian Sebline (Julian Sebline#8476 on Discord)
```

Comment:
This is the wrong place for this (you've added it to the Changelog of 0.4.0). Changelog entries need to go to the end of the first section of the unreleased next version - in this case you should add it to the 0.9.0 section.